apps/predbat/predbat.py

```diff
@@ -43,6 +43,88 @@
 
 INVERTER_TYPES = {"GE": "GivEnergy", "GS": "Ginlong Solis", "SE": "SolarEdge", "SX4": "Solax Gen4 (Modbus Power Control)"}
 
+CONFIG_GROUPS = {
+    "Cheap Night / Bad Export": {
+        "example": "Octopus Go / Economy 7",
+        "description": "In this scenario you will want to charge overnight based on the next days solar forecast.",
+        "items": {
+            "set_soc_enable": True,
+            "set_reserve_enable": True,
+            "set_reserve_hold": True,
+            "calculate_best_charge": True,
+            "combine_charge_slots": True,
+            "metric_min_improvement": 0,
+            "set_charge_window": True,
+            "best_soc_keep": 2.0,
+            "best_soc_min": 0.0,
+            "rate_low_threshold": 0.8,
+            "forecast_plan_hours": 24,
```

Comment:
forecast_plan_hours is in config.yml currently so can't be changed here. We could move it into HA to allow dynamic configuration.

Reply:
I changed forecast_plan_hours to an HA config element now